### DISCUSSION TITLE
How to remove garbage blocks not needed for electronic goods?

### DISCUSSION THREAD

**Discussion by Willsense** [woocommerce/woocommerce#53349] [Checkout Flow]
How to remove garbage blocks not needed for electronic goods? 
<img width="493" alt="Снимок экрана 2024-12-02 в 16 08 33" src="https://github.com/user-attachments/assets/14b2cbb8-d7e8-463a-91ab-9fb7415fdcc1">

**Comment by nielslange** [comment]
Hello @Willsense 👋

Your screenshot appears to highlight the Billing Address block:

<img width="288" alt="Screenshot 2024-12-03 at 11 39 48" src="https://github.com/user-attachments/assets/3208fe3a-17e1-4e65-aced-4884a954d974"><br>

By "electronic goods", are you referring to digital products such as licences, subscriptions, downloads, etc.? If so, I assume you are trying to remove the Shipping Address block instead. If my assumption is correct, all you need to do is to mark your product as a virtual product within the product editor. On [Virtual and downloadable products](https://woocommerce.com/document/managing-products/virtual-downloadable/) you can find detailed information on how to do that.

If a shopper adds a virtual product to the cart, they will no longer see the Shipping Address block on the checkout page. For example, this is how the checkout page looks like when the cart only contains a virtual product:

<img width="1155" alt="Screenshot 2024-12-03 at 11 46 29" src="https://github.com/user-attachments/assets/fba12382-26ef-404c-a9cb-0f5595498aef">

**Reply by Willsense** [reply]
I need to delete Billing address.

**Reply by nielslange** [reply]
Thanks for clarifying this. In https://github.com/woocommerce/woocommerce/discussions/45568#discussioncomment-8828548, @senadir shared steps on how to remove the billing section if needed. I hope this helps.
